migrate/20240821_github_repository_temporary_access_key.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:github_repository) do
+      add_column :session_token, :text, collate: '"C"'
```

Comment:
We can set null:false for that column as the last commit of that PR by refreshing existing tokens during deployment.

Reply:
`access_key` and `secret_key` are also nullable. If Ubicloud cache isn't being used, the GithubRepository doesn't need a session_token